stdlib/sysconfig.pyi

```diff
@@ -14,12 +16,21 @@ __all__ = [
     "parse_config_h",
 ]
 
+if sys.version_info >= (3, 10):
+    __all__ += ["get_default_scheme", "get_preferred_scheme"]
+
```

Comment:
The runtime doesn't include these in `__all__`, so we shouldn't include them in `__all__` in the stub either

```suggestion
```